ProcessLib/LiquidFlow/LiquidFlowMaterialProperties.h

```diff
@@ -44,9 +53,23 @@ class LiquidFlowMaterialProperties
     typedef MaterialLib::Fluid::FluidProperty::ArrayType ArrayType;
 
     LiquidFlowMaterialProperties(
```

Comment:
Is it correct to say: `LiquidFlowMaterialProperties` is the collection of material properties special to the `LiquidFlow` process? Maybe you can add some documentation?